### PR TITLE
SQLQuery1.sql

### PR DESCRIPTION
**Pull request by ColleenWoolsey**
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
1.  Seeding Product side database tables with testable data
2. Added a README.md file with the ERD

# Testing Instructions
Checkout branch initialize-sql
Open it in SSMS
Execute query

Please describe the tests required to verify your changes. Provide instructions so PR Tester can check functionality. Please also list any relevant details for your tests

Check bottom of  SSMS screen for "Query executed"

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] I have added test instructions that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by Daniel-Brewer — APPROVED** [review]
done as group

**Review by dakotacookson — APPROVED** [review]
Aproved

File: README.md [hunk 1]
```diff
@@ -0,0 +1,1586 @@
+
```
Comment (askingalot) [inline README.md:3]: Your readme file should be in markdown, not html.

**Review by askingalot — CHANGES_REQUESTED** [review]
The readme needs to be corrected.

The SQL script looks good.

File: README.md (hunk 1 above)
Comment (ColleenWoolsey) [reply README.md:3]: Created README.md in Markdown

**Review by askingalot — APPROVED** [review]

**Pull request merged**